shell.nix

```diff
@@ -3,14 +3,17 @@
   sha256 = "sha256:1k5nvn2yzw370cqsfh62lncsgydq2qkbjrx34cprzf0k6b93v7ch";
 }) {} }:
 
+
 pkgs.mkShell {
   name = "nim-waku-build-shell";
+  stdenv = pkgs.clangStdenv;
```

Comment:
Do not do this. This way you'll force clang `stdenv` for all other platforms too, including Linux.

Reply:
Look:
```
admin@maci7-03.ms-eu-dublin.ci.devel:~/nwaku % nix repl shell.nix
warning: future versions of Nix will require using `--file` to load a file
Welcome to Nix 2.11.0. Type :? for help.

Loading installable ''...
Added 46 variables.
nix-repl> stdenv    
«derivation /nix/store/zysj3b4rk5dhizhj11l02drdrav2k498-stdenv-darwin.drv»

nix-repl> stdenv.cc
«derivation /nix/store/yi4rbiq8rs2rsbip8za2bl1nsmjmb9dk-clang-wrapper-11.1.0.drv»
```

Reply:
Addressed in [444d310](https://github.com/status-im/nwaku/pull/1190/commits/444d310131d00643fb80cfd90973df187520711e)